serverless.yml

```diff
@@ -24,7 +24,7 @@ provider:
   # TODO: move these to specific functions below once re-arch is done
   environment:
     DEPLOYED: true
-    STAGE: ${opt:stage, self:provider.stage}
+    STAGE: ${sls:stage}
```

Comment:
recently learned that `${sls:stage}` is a built-in magic/shorthand for `${opt:stage, self:provider.stage, "dev"}` (https://www.serverless.com/framework/docs/providers/aws/guide/variables/#referencing-serverless-core-variables)